crates/context/interface/src/journaled_state.rs

```diff
@@ -290,6 +290,53 @@ impl<T> StateLoad<T> {
     }
 }
 
+/// Duplicates StateLoad, adding code cold status.
```

Comment:
@rakita I needed to return `is_code_cold` from account load calls to use it for gas calculations. Is there a better way to extend `StateLoad` without duplicating the rest of the code here? `StateLoad` is widely used and was hesitant to change it